api/models/User.js

```diff
@@ -183,30 +182,28 @@ module.exports = bookshelf.Model.extend(merge({
       - delete their user record!
     */
 
+    // look up their uploads and delete
```

Comment:
Somewhat blocked on what to do here. I am assuming there is something happening outside of the node code that is putting files into a CDN, after they are inserted into s3. And our db records seem to mainly reference the CDN urls. 

So to delete, I have to reconstruct s3 keys from a file's CDN url? Is that possible? And then I have to delete the s3 object and also invalidate the CDN url record?